molecule/default/tests/test_cinder_service.py

```diff
@@ -1,23 +1,30 @@
-import os
+# -*- coding: utf-8 -*-
+# ==============================================================================
+# Imports
+# ==============================================================================
 import pytest
-import testinfra.utils.ansible_runner
-
-testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('shared-infra_hosts')[:1]
-
-utility_container = ("lxc-attach -n $(lxc-ls -1 | grep utility | head -n 1) "
-                     "-- bash -c '. /root/openrc ; ")
+import pprint
 
```

Comment:
Use `unmunchify` to get regular dictionary objects.

```suggestion
from munch import unmunchify
```

Reply:
👍 will update in the next push